learn/framework/xcm.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Native Cross-Chain Communication
+description: Tanssi appchains benefit from XCM, a native cross-chain communication language, which allows fast and secure bridging guaranteed by Polkadot's relay chain.
+---
+
+# Native Cross-Chain Communication
+
+## Introduction {: #introduction }
+
+All the Tanssi appchains have an inherent capability to communicate and interoperate with any other appchain from the Polkadot ecosystem (be it Tanssi-powered or not). This native cross-chain communication feature is possible thanks to the Polkadot relay chain, which guarantees message delivery, and the XCM (Cross-Consensus Message) format, which provides a lingua franca.
```

Comment:
Find a synonym for "lingua franca" feels weird haha.

Also - I would not specifically say "thanks to Polkadot relay chain" to move away from the Polkadot Centric messaging, but more something like "is possible thanks to the architecture Tanssi appchains are built on top off"